donation/report/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove coding utf-8!

Reply:
@YogeshMahera-SerpentCS Yes, but in Check [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1)